.github/workflows/claude.yml

```
@@ -11,12 +11,46 @@ on:
     types: [submitted]
 
 jobs:
+  check-membership:
+    permissions:
+      organization: read
```

actionlint (1.7.7)

unknown permission scope "organization". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)